_includes/json_schema_type.html

```diff
@@ -0,0 +1,26 @@
+
+{% if include.object.items['$ref'] %}
+
+{%- capture type -%}
+<a href="{{include.object.items['$ref'] | replace_first: '/definitions/', ''}}">
+array&lt;{{include.object.items['$ref'] }}&gt;
+</a>
+{% endcapture %}
+{{-type-}}
```

Comment:
why do you first capture it and then directly print it?

Reply:
you're right, I have changed that !